gradle-baseline-java/src/main/groovy/com/palantir/baseline/plugins/BaselineNullAway.java

```diff
@@ -90,4 +102,41 @@ public void execute(JavaCompile javaCompile) {
             }
         });
     }
+
+    // Workaround for nullaway bugs in the last release to support jdk-15. Projects that don't use jdk-15
+    // resolve a newer nullaway version with bug-fixes. This may be deleted after we've rolled everything
+    // off jdk-15 MTS.
+    private static void newerNullAwayInNonJdk15Projects(Project project) {
+        project.getConfigurations()
+                .matching(new Spec<Configuration>() {
+                    @Override
+                    public boolean isSatisfiedBy(Configuration config) {
+                        return "errorprone".equals(config.getName());
+                    }
+                })
+                .configureEach(new Action<Configuration>() {
+                    @Override
+                    public void execute(Configuration _files) {
+                        for (String dep : NON_15_DEPS) {
+                            project.getDependencies().addProvider("errorprone", project.provider(() -> {
+                                if (anyProjectUsesJava15(project)) {
+                                    // Cannot upgrade dependencies in this case because newer nullaway/checkerframework
+                                    // are not compatible with java 15, but fully support jdk11 and jdk17.
+                                    return null;
+                                }
+                                return dep;
```

Comment:
Simplify
```suggestion
                                // Cannot upgrade dependencies in this case because newer nullaway/checkerframework
                                // are not compatible with java 15, but fully support jdk11 and jdk17.
                                return anyProjectUsesJava15(project) ? null : dep;
```